gradle.properties

```diff
@@ -146,7 +146,7 @@ springCloudKubernetesConfig=3.0.1
 springWsVersion=4.0.1
 springKafkaVersion=3.0.2
 kafkaVersion=3.4.0
-wss4jVersion=4.0.0
+wss4jVersion=3.0.0
```

Comment:
4.0.0 is being built from https://github.com/mmoayyed/ws-wss4j/tree/opensamlv5 while waiting for a release.

Reply:
I'll get it bumped back later today.. It was confusing to build something that I couldn't find the jar for.  Thank you for pointing me to where it was.

Reply:
This should be back to 4.0.0.